src/modules/headset/oculus.c

```diff
@@ -308,7 +312,18 @@ static bool oculus_getAxis(Device device, DeviceAxis axis, vec3 value) {
 }
 
 static bool oculus_vibrate(Device device, float strength, float duration, float frequency) {
-  return false; // TODO
+  int idx = device == DEVICE_HAND_LEFT ? 0 : 1;
```

Comment:
Note that this will vibrate the right hand if you do things like `lovr.headset.vibrate('head')`.